_episodes/04-files.md

```diff
@@ -162,11 +162,11 @@ where the maxima are a bit less regular, but the minima are consistently zero.
 > > filenames = glob.glob('inflammation*.csv')
 > > composite_data = numpy.zeros((60,40))
 > >
-> > for f in filenames:
-> >     data = numpy.loadtxt(fname = f, delimiter=',')
+> > for filename in filenames:
+> >     data = numpy.loadtxt(fname = filename, delimiter=',')
 > >     composite_data += data
 > >
-> > composite_data/=len(filenames)
+> > composite_data /= len(filenames)
```

Comment:
@ldko and @annefou, we need to check that we mention compound operators in the introduction

Reply:
We do not mention compound operators in the introduction. But we do it here. To be honest, I sometimes do not mention it and use
composite_data = composite_data + data
etc.
Maybe we should add a note in the instructor guide; what do you think?

Reply:
@nspies, please ignore this discussion when doing the PR split -- we'll address this in a separate PR. You changes are good.